tests/mysql/query.rs

```diff
@@ -790,6 +790,19 @@ fn select_48() {
     );
 }
 
+#[test]
+fn select_49() {
+    assert_eq!(
+        Query::select()
+            .exists()
+            .columns(vec![Char::Character])
+            .from(Char::Table)
+            .and_where(Expr::col(Char::Character).eq("A"))
+            .to_string(MysqlQueryBuilder),
+        "SELECT EXISTS (SELECT * FROM `character` WHERE `character` = 'A') AS `exists`"
+    );
+}
```

Comment:
Btw... do you think we should allow user to customize the alias? Let say someone want `SELECT EXISTS ( ... ) AS found`
```sql
SELECT EXISTS (SELECT * FROM `character` WHERE `character` = 'A') AS `found`
```
We could provide a method `exists_as` that takes any `IntoIden` as input.

Reply:
Yes, this could be added.

Reply:
thinking further about it, I can't see a use case where this could be needed:
- If the user requires only one exists, no need to alias it
- We can't get multiple `.select().exists()` from one query without nesting (example below)
- If the user requires multiple of them from two different table, this could be achieved with some nesting like:
```rust
let characters_exists =
    !format(
        "({})",
        Query::select()
                .exists()
                .from(Char::Table)
                .and_where(Expr::col(Char::Character).eq("A"))
    );

let glyphes_exists =
    !format(
        "({})",
        Query::select()
                .exists()
                .from(Glyph::Table)
                .and_where(Expr::col(Glyph::Aspect).eq(1.23))
    );
 
Query::select()
    .expr_as(
        Expr::cust(
            characters_exists,
            Alias::new('characters_exists')
        )
    )
    .expr_as(
        Expr::cust(
            glyphes_exists,
            Alias::new('glyphes_exists')
        )
    );
```
which should ends up with [this perfectly valid syntax](http://sqlfiddle.com/#!9/d2bf79/4/0):
```sql
SELECT
    (SELECT EXISTS (SELECT * FROM `character` WHERE `character` = 'A') AS `exists`) as `characters_exists`
    (SELECT EXISTS (SELECT * FROM `glyph` WHERE `aspect` = 1.23) AS `exists`) as `glyphes_exists`
```

Reply:
That make sense! tricky loll

Reply:
Really tricky yep 👍🏻

Now that it has been developed, we could add a `multiple_exists` method which would accept a `Vec<(SelectStatement, Iden)>` to simplify this use case and avoid the ugly `format!()` to wrap the expressions into parentheses.